src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,16 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        UserService useService = new UserService();
```

Comment:
Please initialize services you rely on in class variable not to create new object each time method is called